quickwit-config/src/config.rs

```diff
@@ -58,23 +58,19 @@ fn default_listen_address() -> String {
     "127.0.0.1".to_string()
 }
 
+fn default_rest_listen_port() -> u16 {
+    7180
```

Comment:
The default is `7280` 
```suggestion
    7280
```